code/datums/mutable_appearance.dm

```diff
@@ -0,0 +1,18 @@
+// Mutable appearances are an inbuilt byond datastructure. Read the documentation on them by hitting F1 in DM.
+// Basically use them instead of images for overlays/underlays and when changing an object's appearance if you're doing so with any regularity.
+// Unless you need the overlay/underlay to have a different direction than the base object. Then you have to use an image due to a bug.
+
+// Mutable appearances are children of images, just so you know.
+
+/mutable_appearance/New()
+	..()
+	plane = FLOAT_PLANE // No clue why this is 0 by default yet images are on FLOAT_PLANE
```

Comment:
Can we please not do this? It makes mutable_appearances more annoying to use for anything other than floating overlays. You have to manually fuck with the plane if you wanted to, say, use these in human icon code.